tests/virt_autotest/switch_to_ssh_and_install_hypervisor.pm

```diff
@@ -25,13 +25,27 @@ use virt_autotest::utils qw(is_kvm_host is_xen_host);
 
 sub run {
 
+    #enable ssh root access for openSUSE TW in sol console
+    #root access is not permitted by default in TW, see bsc#1173067#c2
+    select_console 'sol', await_console => 1;
+    send_key 'ret' if check_screen('sol-console-wait-typing-ret');
+    if (check_screen('text-login')) {
+        enter_cmd "root";
+        assert_screen "password-prompt";
+        type_password;
+        send_key('ret');
+    }
+    assert_screen "text-logged-in-root";
+    enter_cmd("echo \"PermitRootLogin yes\" > /etc/ssh/sshd_config");
+    enter_cmd("systemctl restart sshd");
```

Comment:
What about a new function called 
enable_root_ssh_login ?
So any host needs it can just call it directly.

Reply:
we have already a permit_root_ssh function in utils which is recently added and can be used in SLE, I thought a little bit and am afraid our 'sol' console is not so friendly to assert_script_run... 

sub permit_root_ssh {
    if (is_sle('<16') || is_leap('<16.0')) {
        my $results = script_run("grep 'PermitRootLogin yes' /etc/ssh/sshd_config");
        if (!$results) {
            assert_script_run("sed -i 's/^PermitRootLogin.*\$/PermitRootLogin yes/' /etc/ssh/sshd_config");
            assert_script_run("systemctl restart sshd");
        }
    }
    else {
        assert_script_run("echo 'PermitRootLogin yes' > /etc/ssh/sshd_config.d/root.conf");
        assert_script_run("systemctl restart sshd");
    }
}

Reply:
Have you tried ?  If you are sure there is problem in sol, could you help develop a sol version of permit_root_ssh ? So it can also be reused.

Reply:
I met problem with assert_script_run() in the past in sol console, I am unable to remember the detail, it is likely the return value is not always correct, so assertion sometimes fails but actually the command is executed fine. 'type_string' or 'enter_cmd' are more stable in sol console. I'll try again and figure out the best approach.

Reply:
Just tried, assert_script_run() does not work in sol console, please see my reply https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13060#issuecomment-899329620. I am going to improve the existing permit_root_ssh function to be workable on sol console.

Reply:
Are you going to add alternatives like type_string or similar for sol ? If that is case, I think this may deserve a new function or even a new module to store functions that can be reused in sol.

Reply:
create a separate permit_root_ssh_in_sol() function in utils.pm. It can be used in our future SLE16 test. Verification tests are in progress.